src/VsExtension/VsEvents/OnBuildPackageRestorer.cs

```diff
@@ -598,5 +594,86 @@ public void Dispose()
             _buildEvents.OnBuildBegin -= BuildEvents_OnBuildBegin;
             _solutionEvents.AfterClosing -= SolutionEvents_AfterClosing;
         }
+
+        #region ILogger implementation
+        public void LogDebug(string data)
```

Comment:
Would it make sense to have a separate logger type? There's no reason to tie it to the restorer is there?